app/org/sagebionetworks/bridge/models/studies/ConsentSignature.java

```diff
@@ -54,33 +68,51 @@ private ConsentSignature(@JsonProperty("name") String name, @JsonProperty("birth
      * @param imageData
      *         image data as a Base64 encoded string, optional
      * @param imageMimeType
-     *         image MIME type (ex: image/png), optioanl
+     *         image MIME type (ex: image/png), optional
      * @return validated ConsentSignature
      * @throws InvalidEntityException
      *         if called with invalid fields
      */
     public static ConsentSignature create(@Nonnull String name, @Nonnull String birthdate, @Nullable String imageData,
-            @Nullable String imageMimeType) throws InvalidEntityException {
-        ConsentSignature sig = new ConsentSignature(name, birthdate, imageData, imageMimeType);
+            @Nullable String imageMimeType, @Nonnull long signedOn) throws InvalidEntityException {
+        ConsentSignature sig = new ConsentSignature(name, birthdate, imageData, imageMimeType, signedOn);
         Validate.entityThrowingException(VALIDATOR, sig);
         return sig;
     }
```

Comment:
4. If create() and createFromJson() now both go through the Builder, we could just test the builder and the validator instead of having duplicate tests for "normal" constructor and JSON constructor.